api-event-files/index.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+$url = 'https://plausible.io/api/event';
+
+$headers = [];
+
+$allowed_headers =
+
+	[
+		'Accept',
+		'Accept-Language',
+		'Connection',
+		'Content-Length',
+		'Content-Type',
+		'Cookie',
+		'Origin',
+		'Referer',
+		'Sec-Ch-Ua',
+		'Sec-Ch-Ua-Mobile',
+		'Sec-Ch-Ua-Platform',
+		'Sec-Fetch-Dest',
+		'Sec-Fetch-Mode',
+		'Sec-Fetch-Site',
+		'User-Agent',
+		'X-Forwarded-Host',
+		'X-Forwarded-Port',
+		'X-Forwarded-Proto',
+		'X-Forwarded-Scheme',
+		'X-Original-Host',
+		'X-Real-Ip',
+		'X-Request-Id',
+		'X-Scheme',
+	];
+
+foreach ( getallheaders() as $key => $value ) {
+	if ( in_array( $key, $allowed_headers ) ) {
+		$headers[ $key ] = sprintf( '%s: %s', $key, $value );
+	}
+}
+
+
+if ( ! empty( $_SERVER['HTTP_CLIENT_IP'] ) ) {
+	$ip = $_SERVER['HTTP_CLIENT_IP'];
+} elseif ( ! empty( $_SERVER['HTTP_X_FORWARDED_FOR'] ) ) {
+	$ip = $_SERVER['HTTP_X_FORWARDED_FOR'];
+} else {
+	$ip = $_SERVER['REMOTE_ADDR'];
+}
+
+$headers['X-Forwarded-For'] = 'X-Forwarded-For: ' . $headers['X-Real-Ip'];
+
+$ch = curl_init( $url );
+
+curl_setopt( $ch, CURLOPT_HTTPHEADER, $headers );
+curl_setopt( $ch, CURLOPT_FOLLOWLOCATION, true );
+curl_setopt( $ch, CURLOPT_HEADER, false );
+curl_setopt( $ch, CURLOPT_RETURNTRANSFER, true );
+curl_setopt( $ch, CURLOPT_SSL_VERIFYHOST, false );
+curl_setopt( $ch, CURLOPT_SSL_VERIFYPEER, false );
+
+
+curl_setopt( $ch, CURLOPT_POST, true );
+curl_setopt( $ch, CURLOPT_POSTFIELDS, file_get_contents( 'php://input' ) );
+
+curl_exec( $ch );
```

Comment:
Is it possible to return the response from the Plausible backend back to the client?

We respond with `202 Accepted` whereas in my testing I get `200 OK` as the response. There may also be useful debugging headers in our response which should ideally be returned back to the browser.

Reply:
what about returning the whole header I'm receiving from curl ?

![image](https://user-images.githubusercontent.com/1303935/203813433-14c7fd2b-60a4-4ce5-929e-087776146b70.png)



![image](https://user-images.githubusercontent.com/1303935/203812909-3cf1a408-e055-47e7-bd09-e36c3ea9b319.png)

Reply:
Yes, ideally the WP plugin would return exactly what it got from plausible.io: same response code, headers and body

Reply:
Is this resolved? Do we now return the exact same response that we get from plausible.io?